feature-utils/poly-analysis/src/analysis/report/ministories-status-analysis.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+
+/**
+ * I am a "fake" analysis that shows the status of running analyses.
+ * I do not have an analyse method as I get the data directly in the constructor.
+ *
+ * I provide the same API as an analysis that should be included in a report.
+ *
+ * @class
+ */
+export default class MinistoriesStatusAnalysis {
```

Comment:
This is mostly the same as inactive-cards-summary. Can be undried to a single "summary" or whatever story. Also, same thing about rendering.

Reply:
We will understand this better when we rework the render and reports